core/shared/src/main/scala/fs2/Scheduler.scala

```diff
@@ -86,16 +97,27 @@ object time {
    * `Scheduler` to signal duration and avoid blocking on thread. After the signal, the execution continues
    * on the supplied execution context.
    */
-  def sleep[F[_]](d: FiniteDuration)(implicit F: Async[F], scheduler: Scheduler, ec: ExecutionContext): Stream[F, Unit] = {
+  def sleep[F[_]](d: FiniteDuration)(implicit F: Async[F], ec: ExecutionContext): Stream[F, Unit] = {
     Stream.eval(F.async[Unit] { cb =>
-      scheduler.scheduleOnce(d) {
+      scheduleOnce(d) {
         ec.execute(() => cb(Right(())))
       }
       ()
     })
   }
 
   /** Identical to `sleep(d).drain`. */
-  def sleep_[F[_]](d: FiniteDuration)(implicit F: Effect[F], scheduler: Scheduler, ec: ExecutionContext): Stream[F, Nothing] =
+  def sleep_[F[_]](d: FiniteDuration)(implicit F: Effect[F], ec: ExecutionContext): Stream[F, Nothing] =
```

Comment:
Sorry to meddle, but since you happen to be modifying this, you might want to replace `Effect` with `Async` here.